clusters/app.ci/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -70,3 +70,44 @@ items:
     - get
     - list
     - watch
+
+- kind: Group
```

Comment:
I don't think the group should be defined in this repo or at most with an unset `users` key

Reply:
Yeah -- othewise `applyconfig` will reset it to this form after each merge to openshift/release,

Reply:
I removed the whole group here. The `group-auto-updater` tool will create it if it doesn't exist.